### ISSUE TITLE
Cannot hook events.remove

### ISSUE THREAD

**Issue by rexdf** [wbond/package_control#952]
follow the script on https://packagecontrol.io/docs/events

```python
import sys

package_name = 'test'

def plugin_loaded():
    from package_control import events

    if events.install(package_name):
        print('Installed %s!' % events.install(package_name))
    elif events.post_upgrade(package_name):
        print('Upgraded to %s!' % events.post_upgrade(package_name))

def plugin_unloaded():
    from package_control import events

    if events.pre_upgrade(package_name):
        print('Upgrading from %s!' % events.pre_upgrade(package_name))
    elif events.remove(package_name):
        print('Removing %s!' % events.remove(package_name))


# Compat with ST2
if sys.version_info < (3,):
    plugin_loaded()
    unload_handler = plugin_unloaded
```

When removing `test`,  cannot find `Removing test` in log.

```log
ignored packages updated to: ["Vintage", "test"]
unloading plugin test.test
reloading Packages/User/Package Control.sublime-settings
reloading Packages/User/Preferences.sublime-settings
ReadDirectoryChangesW fails with ERROR_ACCESS_DENIED
Package Control: The package test has been removed
ignored packages updated to: ["Vintage"]
reloading Packages/User/Package Control.sublime-settings
reloading Packages/User/Preferences.sublime-settings
reloading Packages/User/Preferences.sublime-settings
```

**Comment by FichteFoll** [comment]
Can confirm,

```
ignored packages updated to: ["BracketGuard", "GhostText", "Theme - Aprosopo", "Schemr", "RestructuredText", "Vintage", "Color Highlighter", "Markdown", "Stylus", "test"]
unloading plugin test.test
reloading Packages/User/Package Control.sublime-settings
generating syntax summary
reloading Packages/User/Preferences.sublime-settings
ReadDirectoryChangesW fails with ERROR_ACCESS_DENIED
Package Control: The package test has been removed and will be cleaned up on the next restart
ignored packages updated to: ["Theme - Aprosopo", "BracketGuard", "Stylus", "GhostText", "Schemr", "Vintage", "RestructuredText", "Color Highlighter", "Markdown"]
```
(the directory was in fact deleted despite the error message).

Win8, ST3088, PC 3.1.0-beta

**Comment by wbond** [comment]
@FichteFoll So, what you experienced seems to be functioning "correctly" according to the current codebase. If an error occurs removing the file, the `remove` event is not fired.

This seems like a different issue than @rexdf.

**Comment by wbond** [comment]
I believe the issue here is that there is no version number for that package (unless you bundled it up on GitHub and installed it that way), and thus the `events.remove()` function is returning a version of `None` which does not trigger the `if` block.

**Comment by wbond** [comment]
Alright, after further investigation, I determined it was not that issue, but that the internal Package Control code was using a different in-memory copy of the `events` module than the public API, so the messages were never being received. Fixed by 99e0652c9d676d2d96072fd4ff2cd2224d4c34ca.
